tests/tasks-sqs/build.gradle.kts

```diff
@@ -13,3 +13,8 @@ micronaut {
         additionalModules.add("localstack-sqs")
     }
 }
+configurations.all {
+    resolutionStrategy {
+        force("${libs.aws.sqs.get()}")
```

Comment:
The tests here pass without this, is that expected?

Reply:
It fails for me locally https://ge.micronaut.io/s/4lgnag3g2cfca